tests/robust/test_ops.py

```diff
@@ -141,3 +142,9 @@ def test_influence_raises_no_grad_warning_correctly():
 
     with pytest.warns(UserWarning, match="torch.no_grad"):
         influence()
+
+    with pytest.warns() as record:
+        with torch.no_grad():
+            influence()
+        assert len(record) == 0
+        warnings.warn("Dummy warning.")
```

Comment:
What is this dummy warning for?

Reply:
Because `with pytest.warns()` requires a warning to pass. I first check no warnings are thrown in the assert. This is a workaround b/c `pytest.warns(None)` is no longer supported.